eslint.config.js

```diff
@@ -0,0 +1,58 @@
+import { defineConfig } from "eslint-define-config";
+import vuePlugin from "eslint-plugin-vue";
+import prettierPlugin from "eslint-plugin-prettier";
+import vueParser from "vue-eslint-parser";
+
+export default defineConfig([
+    {
+        files: ["**/*.js", "**/*.vue"],
+        languageOptions: {
+            ecmaVersion: "latest",
+            sourceType: "module",
+            globals: {
+                cordova: "readonly",
+                cordovaUI: "readonly",
+                ol: "readonly",
+                wNumb: "readonly",
+                ConfigStorage: "readonly",
+                // start cordova bindings, remove after cordova is removed/replace/modularized
+                cordova_serial: "readonly",
```

Comment:
We have Cordova things? I think remember we removed anything Cordova related...

Reply:
@McGiverGim 
- This PR only migrates the existing configuration
- Removing remnants of cordova should be done in separate PR